Backend/src/config/passport-setup.ts

```diff
@@ -0,0 +1,61 @@
+import passport from 'passport';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import { type users } from '@prisma/client'; // Make sure to import the User type from Prisma
+import keys from './keys';
+import UserService from '@/modules/users/users.service';
+
+passport.serializeUser((user: users, done) => {

```

Comment:
`user: users` is it user or users?

Reply:
it is users, It comes from prisma:
(alias) type users = {
    id: string;
    email: string;
    name: string;
    phone: string | null;
    profilePicture: string | null;
    country: string | null;
    university: string | null;
    language: string | null;
}
import users